src/plugins/visualizations/public/plugin.ts

```diff
@@ -214,14 +216,6 @@ export class VisualizationsPlugin
       chrome: core.chrome,
       overlays: core.overlays,
     });
-    const savedAugmentVisLoader = createSavedAugmentVisLoader({
-      savedObjectsClient: core.savedObjects.client,
-      indexPatterns: data.indexPatterns,
-      search: data.search,
-      chrome: core.chrome,
-      overlays: core.overlays,
-    });
-    setSavedAugmentVisLoader(savedAugmentVisLoader);
```

Comment:
We will still need this - why have you removed?

Reply:
brought it back